### DISCUSSION TITLE
why can't I see my calcite twins?

### DISCUSSION THREAD

**Discussion by macosta541** [mtex-toolbox/mtex#1433] [Ask Anything] [answered]
Hi all,

Maybe this is a general EBSD question more than an MTEX -specific question, so apologies if this is inappropriate for this forum (is there a more suitable EBSD forum out there?) . 

I'm still learning the technique and so maybe this is a stupid question.

I cannot see the calcite twins in my raw EBSD data.  I attach an image of calcite ebsd data, and as you can see there are no twins discernable. Shouldn't I be able to see them (at least some of the grains have twins in thin section)?  Was the data acquired incorrectly?  

The data was acquired on a Tescan Mira field emission SEM with an Oxford Instruments Symmetry detector using the Aztec software.

![image](https://user-images.githubusercontent.com/74112001/169478424-20b3ae97-98da-470d-a835-0c6edc7d62c0.png)

**Comment by kilir** [comment]
Hi,
could you add a little more info, especially with what material file you indexed your calcite as well as what we are looking at in the picture you show.
Cheers,
Rüdiger

**Reply by macosta541** [reply]
Hi,

Yes, from the .cpr file:

StructureName=Calcite
Reference=[Calcite.cry]
Enabled=True
a=4.98999977111816
b=4.98999977111816
c=17.0639991760254
alpha=90
beta=90
gamma=120
LaueGroup=7
SpaceGroup=0
ID1=
ID2=
NumberOfReflectors=117

The picture is the IPF of calcite in the cleaning software (aztecHKL)

If I take the data and run it through MTEX, this is what I get for the grain reconstructions and orientations of the calcite (I didn't plot any of the other phases)

![image](https://user-images.githubusercontent.com/74112001/169525492-b18701fb-b091-412c-a337-3f57b51c18eb.png)

(*the wonky area in the mid-right hand part of the map is where the beam stopped for whatever reason and the reconstruction doesn't know what to do about it)

When I plot the neighbor-pair misorientations for calcite-calcite, this is what I get (the twins aren't obviously there either)

![image](https://user-images.githubusercontent.com/74112001/169526042-3bbcb4f6-364e-4e1c-a52a-cdcf095e6390.png)

**Reply by kilir** [reply]
Could you look at a 1:1 comparison LM/EBSD? Also, are your twins wide enough to be picked up at your step size?
Cheers,
Rüdiger

**Comment by macosta541** [comment, marked as answer]
Okay, here's what I ended up concluding:

I think that the step size used to acquire the data was too large to resolve twins. According to Lacombe et al. (2021), you need a step size of about 0.5 microns to resolve them.  So, I will probably choose a few representative grains to measure with higher resolution and then call it good since all I am interested in is whether or not they are growth or deformation twins.



Lacombe, O. (2010). Calcite twins, a tool for tectonic studies in thrust belts and stable orogenic forelands. Oil & Gas Science and Technology–Revue d’IFP Energies nouvelles, 65(6), 809-838.
